src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -79,5 +79,10 @@ public static class CustomizedPingTimer
             public const string ServiceStatus = "ServiceStatus";
             public const string Servers = "Servers";
         }
+
+        public static class Protocol
+        {
+            public const string BlazorPack = "blazorpack";
```

Comment:
this is a little bit tricky, is there another way instead of depending on hardcoding blazor's protocol?

Reply:
Come up by another fix on the supported protocol: https://github.com/Azure/azure-signalr/issues/693, I think protocol is something public and stable to use. Another choice is to detect DI related components, also hard code and it varies in different versions of Blaozr.

Reply:
yeah, that other fix is planned to be removed for net5 though....